### DISCUSSION TITLE
New columns on standard objects

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#7816] [Ideas]
# company

## industry

see old issue: https://github.com/twentyhq/twenty/issues/3048

## employeesRange
- 1-10
- 11-50
- 51-250
- 251-1K
- 1K-5K
- 5K-10K
- 10K-50K
- 50K-100K
- 100K+

## annualRevenue 
Step 1: rename annualRecurringRevenue to annualRevenue
Step 2: disable this field by default for newly created workspace
Step 3: create a new field, enabled by default on all new workspaces, called annualRevenueRange, of type enum, with values:

$0-$1M
$1M-$10M
$10M-$50M
$50M-$100M
$100M-$250M
$250M-$500M
$500M-$1B
$1B-$10B
$10B+

## description

A brief summary/description of the company.
Short text (typically a paragraph).

Enabled by default but not visible on default view.

## founded
year

## funding 

Need to add 3 fields, disabled by default:
- last_funding_date latest_funding_stage (d/m/y)
- total_funding_raised (currency field)
- latest_funding_stage ([values](https://docs.peopledatalabs.com/docs/funding-rounds))

## related companies
many-to-many

# person

## Gender
male/female

## previous experiences
many-to-many with people
## Language
Array of values coming from an enum.
Disabled by default.
Take values from:
https://docs.peopledatalabs.com/docs/language-names

**Comment by charlesBochet** [comment]
Same comment as: https://github.com/twentyhq/twenty/issues/2977

**Comment by charlesBochet** [comment]
We have introduced all the tooling required to handle this one.
Ready to be picked

**Comment by 16dango** [comment]
I'll try to work on, please assign me, sir🙇

**Comment by FelixMalfait** [comment]
Thanks!

**Comment by pacyL2K19** [comment]
I can work on this as well @Bonapara @FelixMalfait

**Comment by FelixMalfait** [comment]
Thanks a lot @pacyL2K19 I removed the "good first issue" tag as I think we should probably tackle all field updates at the same time (they are several related issues).

**Comment by charlesBochet** [comment]
@FelixMalfait should we close this ticket?

**Comment by FelixMalfait** [comment]
@charlesBochet I'd like to bring it back when we consider data enrichment. It's not that useful I agree but there are a few things that I would re-use / based on analysis I did when looking at various data enrichment providers. I closed most related issues

**Comment by Kausol** [comment]
Thanks!

**Comment by BOHEUS** [comment]
@FelixMalfait is it still relevant and if so, maybe it'd be better to extend existing Apollo enrichment app with it?

**Reply by FelixMalfait** [reply]
Yes probably not very relevant. We still want to update the standard object fields which are pretty bad (ICP etc), but will keep it to a minimal and most fields should live in apps yes!
